protected/humhub/widgets/MetaSearchWidget.php

```diff
@@ -65,4 +66,30 @@ public function addProvider($provider)
     {
         $this->providers[] = $provider;
     }
+
+    /**
+     * @return MetaSearchProviderInterface[]
+     */
+    protected function getSortedProviders(): array
```

Comment:
private?

Reply:
@luke- Yes, it may be updated to `private` without any problems for now, but I did it as `protected` for future case if the class will be used as parent class from some external module so it would be good allow to run `$this->getSortedProviders()` from a child class.
But if you think we don't need this I can make it private, just please tell me, thanks.

Reply:
@yurabakhtin Let's make it "private" for now. When we need it we can change the visibility to "protected".